src/abstractdataframe/io.jl

```diff
@@ -387,6 +410,16 @@ function Base.show(io::IO, mime::MIME"text/tab-separated-values", dfr::DataFrame
     show(io, mime, view(parent(dfr), [r], c))
 end
 
+function Base.show(io::IO, mime::MIME"text/csv",
```

Comment:
Use short function syntax?

Reply:
fixed (also in other places in the file)